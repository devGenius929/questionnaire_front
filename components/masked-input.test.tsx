import React from 'react';
import renderer from 'react-test-renderer';
import MaskedInput from './masked-input';

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <MaskedInput
          id="dadosbasicos-titular-titulo"
          mask="999.999.999-99"
          name="voterRegistration"
          className="tituloEleitor"
          value="something"
          placeholder="CPF"
          onChange={(e) => console.log('Changed!')}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
