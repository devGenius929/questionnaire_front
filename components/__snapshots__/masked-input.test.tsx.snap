// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<input
  className="tituloEleitor"
  id="dadosbasicos-titular-titulo"
  name="voterRegistration"
  onChange={[Function]}
  placeholder="CPF"
  type="text"
  value="something"
/>
`;
